command/agent/agent.go

```diff
@@ -891,7 +891,8 @@ func (a *Agent) setupServer() error {
 		rpcServ := &structs.Service{
 			Name:      a.config.Consul.ServerServiceName,
 			PortLabel: a.config.AdvertiseAddrs.RPC,
-			Tags:      append([]string{consul.ServiceTagRPC}, a.config.Consul.Tags...),
+			Tags: append([]string{consul.ServiceTagRPC, a.config.Region},
+				a.config.Consul.Tags...),
```

Comment:
@schmichael what do you think of adding the region tag to the other 2 services too? I can't think of any reason not to, and it'd be nice for consistency when people query.